impl/spanner/directory/directory.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package directory reads and writes Key Transparency Directory information.
+package directory
+
+import (
+	"context"
+	"time"
+
+	"cloud.google.com/go/spanner"
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/any"
+	"github.com/google/keytransparency/core/directory"
+	"github.com/google/trillian/crypto/keyspb"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	ktspanner "github.com/google/keytransparency/impl/spanner"
+	tpb "github.com/google/trillian"
+)
+
+const table = "Directories"
+
+var directoriesCols = []string{"DirectoryID", "Map", "Log", "VRFPublicKey", "VRFPrivateKey", "MinInterval", "MaxInterval", "Deleted"}
```

Comment:
Consider putting those constants to a separate `schema.go` or `sdl.go` file.

Also, how about calling them `dirTable` and `dirColumns`?